internal/context/context.go

```diff
@@ -103,34 +105,44 @@ type UdmNFContext struct {
 func InitUdmContext(context *UDMContext) {
 	config := factory.UdmConfig
 	logger.UtilLog.Info("udmconfig Info: Version[", config.Info.Version, "] Description[", config.Info.Description, "]")
+
 	configuration := config.Configuration
 	udmContext.NfId = uuid.New().String()
 	sbi := configuration.Sbi
-	udmContext.UriScheme = ""
-	udmContext.SBIPort = factory.UdmSbiDefaultPort
-	udmContext.RegisterIPv4 = factory.UdmSbiDefaultIPv4
-	if sbi != nil {
```

Comment:
Is sbi guaranteed to not be nil?
It's good to avoid the long "if nesting" if it is!